src/daemon/bitcoind/poller.rs

```diff
@@ -1536,6 +1536,16 @@ fn maybe_create_wallet(revaultd: &mut RevaultD, bitcoind: &BitcoinD) -> Result<(
         }
         log::trace!("Importing unvault descriptors '{:?}'", &addresses);
         bitcoind.startup_import_unvault_descriptors(addresses, wallet.timestamp, fresh_wallet)?;
+
+        if revaultd.our_man_xpub.is_some() {
+            // Importing the CPFP addresses as well
+            let mut addresses = revaultd.all_cpfp_addresses();
+            for i in 0..addresses.len() {
+                addresses[i] = bitcoind.addr_descriptor(&addresses[i])?;
```

Comment:
Contributing `addr()` descriptors to `rust-miniscript` (in addition to the existing `Bare`) would be a nice performance improvement